compiler/src/iree/compiler/Codegen/SPIRV/KernelConfig.cpp

```diff
@@ -1060,6 +1057,16 @@ static LogicalResult setReductionConfig(const spirv::TargetEnv &targetEnv,
   tileSizes.emplace_back(std::move(workgroupTileSizes));  // Workgroup level
   tileSizes.emplace_back(std::move(reductionTileSizes));  // reduction level
 
+  for (Operation *userOp : op->getUsers()) {
+    if (auto fusedOp = dyn_cast<linalg::LinalgOp>(userOp)) {
+      // Set lowering configuration to drive tiling for fused ops too---the
+      // pipeline expects it.
+      fusedOp->setAttr(
+          getWarpReductionFusedOpTileSizeAttrName(),
```

Comment:
Why is this a different attr name than lowering config? It is encoding the same kind of information. I’m not sure it is better to start adding more attribute names.

Reply:
I think this is the conclusion we had in the discussion previously with @MaheshRavishankar? I don't have strong opinions as whether to reuse existing ones / create new ones. If you'd like to stick to the current lowering configuration I'm fine with that too. Just let me know.

Reply:
I see, I had understood it differently. I thought we were going to use the existing lowering config attribute, we were just going to leave the first level if tiling empty for non-root ops.

Reply:
Yeah. My understanding is that the lowering config really signals a "root op". So attaching that to other "non-root ops" can be confusing in its own way.  Given they are the same, we also need to have logic for example to make sure they are consistent (w.r.t. missing/non-missing parts etc.). Though I agree with you having extra attributes aren't a beautiful solution too. So meh; I don't have a strong opinions towards this. :)

Reply:
There are two solutions.
1) Use the same attribute. Then the same attribute will be used on different ops, but relevant information (like tile factors for tile and fuse) will be present in only the lowering config of one op. This will avoid having information spread across ops and keeping them consistent
2) add a new attribute. So its an add on to whatever is there today. 

Both aren't great solutions. Just in terms of "invasive"-ness while we wait for transform dialect being flushed out the second is better. What I would draw the line on is exposing these to a search. That's where things go downhill fast.
What would be good to keep track is what is the space of values for these attributes for each op. If the same value is used all the time (or can be derived in a similar manner just based on target information), then we can just drop the attribute all together and just compute the values as needed. That will also be more easy to port over to the new approach.

Reply:
My preference is for solution 1. As this is already what cuda path does and if we need to do it for other strategy we don’t need to add more attributes. 
Not a very strong opinion though.

Reply:
Okay. I changed to stick with lowering config attribute and removing the first level of info.